preparations.html

```diff
@@ -74,76 +65,57 @@ <h3>Section Two - DSiWare Game</h3>
-            <li>Open Data Management (Green option in the bottom right)</li>
-            <li>Open DSi Ware</li>
-            <li>Under System Memory, tap your game</li>
-            <li>Tap Copy</li>
-            <li>You will be asked if you are sure to copy it as it will make the game unplayable - Tap Yes</li>
-            <li>Exit out of System Settings and Power off your 3DS</li>
+            <li>Power on your 3DS</li>
+            <li>Open System settings, navigate to Data Management > DSi Ware</li>
+            <li>Under System Memory, tap your game and tap copy</li>
+            <li>Exit out of System settings and power off your 3DS </li>
         </ol>
 
     </div>
     <div>
         <a name="section4"></a>
-        <h3>Section Four - Finding your correct ID0</h3>
+        <h3>Section Four - Getting the ID0 and the dsiware.bin</h3>
         <p>
             We will need your ID0, which is a unique String to identify your console.<br/>
             To get it, please do the following:
         </p>
             <ol>
                 <li>Power off your 3DS if you haven't already</li>
-                <li>Put your SD Card into your PC</li>
-                <li>Open the SD Card's Contents</li>
-                <li>Open the Folder Named <code>Nintendo 3DS</code></li>
-                <li>You will now find one or multiple folders with 32-character-long names.<br/>
-                    <div class="alert alert-success">
-                        If you only have one, that's good! Continue with step 15
-                    </div>
+                <li>Inert your SD Card into your PC and open the <code>Nintendo 3DS</code> folder</li>
+                <li>You will now find one or multiple folders with 32-character-long names
+                <li>If you only have one, that's good! Continue with step 5</li>
+                <ul type=circle>
+                    <li>Navigate one folder up so you can see the <code>Nintendo 3DS</code> folder again</li>
+                    <li>Rename your <code>Nintendo 3DS</code> folder to <code>Nintendo BACKUP</code> by right-clicking it and selecting Rename...</li>
+                    <li>Put the SD Card back into your system and power it on</li>
+                    <li><strong>Wait</strong> for it to generate its data on your SD Card</li>
+                    <li>Power off your system and insert the SD Card back into your PC</li>
+                    <li>Open the <code>Nintendo 3DS</code> folder</li>
+                </ul>
+                <li>You should now see one folder with a 32-character-long name. This name of the folder is your ID0 - remember/copy it</li>
+                <ul type=circle>
+                    <li>If you did <strong>NOT</strong> skip step 4, delete the <code>Nintendo 3DS</code> folder and rename the <code>Nintendo BACKUP</code> folder back to <code>Nintendo 3DS</code></li>
+                </ul>
+                <li>Open its sub-folder with the ID0 you copied</li>

```

Comment:
This instruction is unclear, please rephrase it.

Reply:
What's unclear about it?

Reply:
It only mentions the id0 "subfolder", but there is one more layer.
Should be ``Open the subfolder named like your id0\n Open the next subfolder with a long name``

Reply:
`Navigate into [Your ID0] > 32-character-long name > Nintendo DSiWare` ? Including the DSiWare folder to cut out unnecessary steps 

(or  [Your ID0]\32-character-long name\Nintendo DSiWare since we haven't made an agreement on which to use yet)